ariadne/contrib/federation/schema.py

```diff
@@ -26,26 +26,48 @@
         _service: _Service!
     }}
 
-    directive @external on FIELD_DEFINITION
-	directive @requires(fields: String!) on FIELD_DEFINITION
-	directive @provides(fields: String!) on FIELD_DEFINITION
-	directive @extends on OBJECT | INTERFACE
+    directive @extends on OBJECT | INTERFACE
+    directive @external on OBJECT | FIELD_DEFINITION
+    directive @requires(fields: FieldSet!) on FIELD_DEFINITION
+    directive @provides(fields: FieldSet!) on FIELD_DEFINITION
+    directive @tag(name: String!) repeatable on FIELD_DEFINITION | INTERFACE | OBJECT | UNION | ARGUMENT_DEFINITION | SCALAR | ENUM | ENUM_VALUE | INPUT_OBJECT | INPUT_FIELD_DEFINITION
+    scalar FieldSet
```

Comment:
Federation v1 actually uses `_FieldSet` scalar, it was renamed to `FieldSet` in v2 (guessing to avoid weird naming when namespacing it which would result in two underscores, i.e. `federation__FieldSet` vs `federation_FieldSet`)